user_docs/en/userGuide.t2t

```diff
@@ -1709,8 +1709,8 @@ This field sets the amount of lines you will move by when pressing page up or pa
 ==== Use screen layout ====[BrowseModeSettingsScreenLayout]
 Key: NVDA+v
 
-This option allows you to specify whether content in browse mode should place content such as links and other fields on their own line, or if it should keep them in the flow of text as it is visually shown.
-If the option is enabled then things will stay as they are visually shown, but if it is disabled then fields will be placed on their own line.
+This option allows you to specify whether browse mode should place clickable content (links, buttons and fields) on its own line, or if it should keep it in the flow of text as it is visually shown. Note that this only applies to browse mode as it is used in web browsers. Apps such as Microsoft Word and Outlook are unaffected by this option, and always use screen layout.

```

Comment:
Given that screen layout also applies to the PDF content in Acrobat, all Electron based apps and mails in Thunderbird wouldn't it be better to say that it does not apply for Word and Outlook rather than saying that it applies only in browsers?

Reply:
> Given that screen layout also applies to the PDF content in Acrobat, all Electron based apps and mails in Thunderbird wouldn't it be better to say that it does not apply for Word and Outlook rather than saying that it applies only in browsers?

Yes it would. I have amended it to say that it does not apply to MS office apps such as Word and Outlook.